.travis.yml

```diff
@@ -0,0 +1,50 @@
+language: android
+jdk: oraclejdk8
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+    - $HOME/.android/build-cache
+
+branches:
+  only:
+  # Build only for master PR or Commit
+  - master
+
+env:
+ global:
+    # Global variables to get things easier to change
+    - ANDROID_API_LEVEL_TARGET=28
+    - ANDROID_API_LEVEL_EMULATOR=22
```

Comment:
Your API levels are too high. Our minimum requirement is android 5.0 (API level 21)

Reply:
@xiamaz do you mean because the emulator API ?

Reply:
@xiamaz do you mean because of the Emulator API ??

Reply:
if you check the build.gradle file you see that compiledSdkVersion is 28,  thats why you install API 28 and it works for 21-28. The minimum its also specified there